bin/create

```diff
@@ -65,3 +75,16 @@ chmod 755 "$ROOT/usr/sbin/policy-rc.d"
 
 # workaround bug in livecd-rootfs
 chmod 1777 "$ROOT/tmp"
+
+# tell the user if classic does not match the ubuntu core release
+if ! grep -q 'VERSION_ID=\"16\"' /etc/os-release; then
```

Comment:
AFAIK This will never be true in our setup. As snap that is built against base: core or core16 will keep seeing VERSION_ID="16" here. You need to look at /var/lib/snapd/hostfs/lib/os-release AFAIK